### ISSUE TITLE
[Feature Request][`fix_includes.py`] Add option to completely _disable_ sorting of includes

### ISSUE THREAD

**Issue by Jacobfaib** [include-what-you-use/include-what-you-use#1480]
TL;DR:
------
Is there a way to _completely_ disable sorting of inclusions and forward-declares?

The Problem:
------------
Even with `--noreorder` `fix_includes.py` still applies _some_ sorting of inserted headers. This interferes with other formatters (e.g. `clang-format`) which apply other sorting rules.

This is problematic when both are used in a lint pipeliner such as `pre-commit`:

1. start `pre-commit` which runs `clang-format`....
1. `clang-format` runs, putting the header files in a particular order.
1. `fix_includes.py` runs, and possibly inserts new headers, but re-orders all headers to a new order.
2. `pre-commit` fails, OK do a new `pre-commit` run
3. `goto 2.`

It would be great if there was some option to completely disable header sorting in `fix_includes.py`, as this would break the vicious cycle above.

**Comment by kimgr** [comment]
Yeah, it would be nice to leave existing lines alone. But new lines still need to be injected somewhere?

I'm not sure I follow your precommit pipeline exactly, but couldn't you run `clang-format` last?

**Comment by Jacobfaib** [comment]
The way pre-commit works is that it runs a set of checks before each `git commit`, and if those checks fail (or they make changes for you), then the `git commit` fails and you have to run the whole check pipeline again.

So it doesn't actually matter in what order you have the two.

**Comment by kimgr** [comment]
I see, thanks. I was looking at the move-span/reorder-span code the other day for a similar bug, and I'm actually still not sure I understand all the subtleties -- the spans appear to be sorted for algorithmic reasons too, not just for presentation. Need to read up more before I can make any real judgement. But yeah, I see that the current behavior is untenable.
